realm/realm-library/src/androidTest/java/io/realm/RealmObjectSchemaTests.java

```diff
@@ -43,9 +61,8 @@ protected void setUp() throws Exception {
         schema = realmSchema.create("NewClass");
     }
 
-    @Override
-    protected void tearDown() throws Exception {
-        super.tearDown();
+    @After
+    public void tearDown() throws Exception {
```

Comment:
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a temporary error. The following address(es) deferred:

  jcjacksonengineering@gmail.com
    Domain joe-jackson.net has exceeded the max emails per hour (108/100 (108%)) allowed.  Message will be reattempted later

------- This is a copy of the message, including all the headers. ------
Received: from github-smtp2-ext6.iad.github.net ([192.30.252.197]:32947 helo=github-smtp2b-ext-cp1-prd.iad.github.net)
	by rs5.websitehostserver.net with esmtps (TLSv1.2:ECDHE-RSA-AES256-GCM-SHA384:256)
	(Exim 4.86_1)
	(envelope-from <noreply@github.com>)
	id 1aflj5-001fci-8j
	for jcjackson@joe-jackson.net; Tue, 15 Mar 2016 05:57:19 -0400
Date: Mon, 14 Mar 2016 09:18:10 -0700
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=github.com;
	s=pf2014; t=1457972290;
	bh=SGUnlyhVXk3OeOSzwDLxb6wn5xzhSOB/C7rSAK9Oby8=;
	h=From:Reply-To:To:In-Reply-To:References:Subject:List-ID:
	 List-Archive:List-Post:List-Unsubscribe:From;
	b=eCfx9kIA7cwcaYMcrZFZgxkb0fC3gkWLf7mStqWq9u2S1zGcH7mYeLsuuXgKUtUQx
	 bc+hbeog2HYuYzCEFSF7mXsG7aA8QlqT6DBNDAegrFEj0sbp3VSbPtM9M4sV8Xpweq
	 c/uvbojYoh69UixAXfuV1/3d2wSqiIpI5HORGjeg=
From: Christian Melchior <notifications@github.com>
Reply-To: realm/realm-java <reply@reply.github.com>
To: realm/realm-java <realm-java@noreply.github.com>
Message-ID: <realm/realm-java/pull/2445/r56027335@github.com>
In-Reply-To: <realm/realm-java/pull/2445@github.com>
References: <realm/realm-java/pull/2445@github.com>
Subject: Re: [realm-java] isPrimaryKey/getPrimaryKey on RealmObjectSchema
 (#2445)
Mime-Version: 1.0
Content-Type: multipart/alternative;
 boundary="--==_mimepart_56e6e44245df7_6da53fa8cf5f12b812599c6";
 charset=UTF-8
Content-Transfer-Encoding: 7bit
Precedence: list
X-GitHub-Sender: cmelchior
X-GitHub-Recipient: jcj52436999
List-ID: realm/realm-java <realm-java.realm.github.com>
List-Archive: https://github.com/realm/realm-java
List-Post: <mailto:reply@reply.github.com>
List-Unsubscribe: <mailto:unsub+000f94f77a0b390377da431193753372ff1e4487c156c46e92cf0000000112fea64292a170ce0356e8c7@reply.github.com>,
 <https://github.com/notifications/unsubscribe/AA-U95mU9uwKyKfzHi98CA5t3pUdQnXkks5ptYpCgaJpZM4Hv5lo>
X-Auto-Response-Suppress: All
X-GitHub-Recipient-Address: jcjackson@joe-jackson.net


----==_mimepart_56e6e44245df7_6da53fa8cf5f12b812599c6
Content-Type: text/plain;
 charset=UTF-8
Content-Transfer-Encoding: 7bit

> @@ -43,9 +61,8 @@ protected void setUp() throws Exception {
>          schema = realmSchema.create("NewClass");
>      }
>  
> -    @Override
> -    protected void tearDown() throws Exception {
> -        super.tearDown();
> +    @After
> +    public void tearDown() throws Exception {

No longer throws Exception

---
Reply to this email directly or view it on GitHub:
https://github.com/realm/realm-java/pull/2445/files#r56027335
----==_mimepart_56e6e44245df7_6da53fa8cf5f12b812599c6
Content-Type: text/html;
 charset=UTF-8
Content-Transfer-Encoding: 7bit

<p>In <a href="https://github.com/realm/realm-java/pull/2445#discussion_r56027335">realm/realm-library/src/androidTest/java/io/realm/RealmObjectSchemaTests.java</a>:</p>
<pre style='color:#555'>&gt; @@ -43,9 +61,8 @@ protected void setUp() throws Exception {
&gt;          schema = realmSchema.create(&quot;NewClass&quot;);
&gt;      }
&gt;  
&gt; -    @Override
&gt; -    protected void tearDown() throws Exception {
&gt; -        super.tearDown();
&gt; +    @After
&gt; +    public void tearDown() throws Exception {
</pre>
<p>No longer throws Exception</p>

<p style="font-size:small;-webkit-text-size-adjust:none;color:#666;">&mdash;<br>Reply to this email directly or <a href="https://github.com/realm/realm-java/pull/2445/files#r56027335">view it on GitHub</a>.<img alt="" height="1" src="https://github.com/notifications/beacon/AA-U95Vw1_9B44gV2GuF7CWnNcaHW7KDks5ptYpCgaJpZM4Hv5lo.gif" width="1" /></p>
<div itemscope itemtype="http://schema.org/EmailMessage">
<div itemprop="action" itemscope itemtype="http://schema.org/ViewAction">
  <link itemprop="url" href="https://github.com/realm/realm-java/pull/2445/files#r56027335"></link>
  <meta itemprop="name" content="View Pull Request"></meta>
</div>
<meta itemprop="description" content="View this Pull Request on GitHub"></meta>
</div>

----==_mimepart_56e6e44245df7_6da53fa8cf5f12b812599c6--